contentcuration/contentcuration/utils/celery/tasks.py

```diff
@@ -263,7 +256,7 @@ def fetch_or_enqueue(self, user, **kwargs):
         # create an advisory lock to obtain exclusive control on preventing task duplicates
         with self._lock_signature(signature):
             # order by most recently created
-            task_ids = self.find_incomplete_ids(signature).order_by("-date_created")[:1]
+            task_ids = self.find_incomplete_ids(signature).order_by("-date_created")[:1].values_list("task_id", flat=True)
```

Comment:
So when `find_incomplete_ids` is updated to return `.values_list("task_id", flat=True)` you can remove the trailing bit